### PR TITLE
Adding 3scale and clean up files

### LINKED ISSUE #8: Add 3Scale API Gateway Template
3Scale API Gateway Template: https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_examples/files/examples/v1.5/quickstart-templates/apicast-gateway-template.yml

### PR DESCRIPTION
**Pull request by coreydaley**
removing legacy index.md files

Fixes #8 and #9

### CODE REVIEW

**Comment by coreydaley** [comment]
@bparees @luciddreamz ptal

**Comment by bparees** [comment]
lgtm

File: official.yaml [hunk 1]
```diff
@@ -399,3 +399,10 @@ data:
       - location: https://raw.githubusercontent.com/redhat-developer/s2i-dotnetcore/master/dotnet_imagestreams.json
         docs: https://github.com/redhat-developer/s2i-dotnetcore/blob/master/README.md
         suffix: rhel7
+  cloudforms:
+    templates:
+      - location: https://raw.githubusercontent.com/openshift/openshift-ansible/master/roles/openshift_examples/files/examples/v1.5/cfme-templates/cfme-template.yaml
+  3scale:
+    templates:
+      - location: https://raw.githubusercontent.com/openshift/openshift-ansible/master/roles/openshift_examples/files/examples/v1.5/quickstart-templates/apicast-gateway-template.yml
```
Comment (luciddreamz) [inline official.yaml]: Remove CloudForms. I found the docs on CloudForms, and these appear to be more of "infrastructure" templates, and those two other PV templates that were in the cfme directory appear to be required. Docs: https://access.redhat.com/documentation/en/red-hat-cloudforms/4.2/paged/installing-red-hat-cloudforms-on-openshift-container-platform/

For 3Scale, I just saw their doc on this template. Change it over to:

    3scale:
      templates:
        - location: https://raw.githubusercontent.com/3scale/rhamp-openshift-templates/1.0.0.GA/apicast-gateway/apicast-gateway-template.yml
          docs: https://support.3scale.net/docs/deployment-options/apicast-openshift

**Comment by coreydaley** [comment]
@luciddreamz Requested changes have been implemented.

**Comment by luciddreamz** [comment]
Looks good

**Pull request merged**